includes/class-health-check-debug-data.php

```diff
@@ -523,9 +524,16 @@ static function debug_data( $locale = null ) {
 				$plugin_version_string = sprintf( __( 'Version %s', 'health-check' ), $plugin_version );
```

Comment:
I like more to sanitize the string and the replacement outside the translate function. Also a hint for the translators world be fine, like `// Translators: %s is a username.`
see https://gist.github.com/bueltge/49c652187ef10a448bfb

Reply:
Could you elaborate a bit more please as I got confused on this review, so I can update the commit?

The code is

```
// translators: %s: Plugin version number.
$plugin_version_string = sprintf( __( 'Version %s', 'health-check' ), $plugin_version );
```

I actually made it that way to be in-line with all the other translation notices etc in the file. I'm not sure that I understand what I have to change / if I have to change something.

Reply:
Yes, weldome.
I mean we should always sanitize the input from translation files, in this context replace the `__` with `esc_html__` as example. However this needs to be switch the `%s` out side the translation function so that we should sanitize the variable.
```php
// translators: %s: Plugin version number.
$plugin_version_string = sprintf( __( 'Version %s', 'health-check' ), esc_html( $plugin_version ) );
```

Reply:
Ah ok now I understand. Thanks for the update 👍 

The thing here is that `$plugin_version` comes from `get_plugins()` which is a core function. I don't see a point on sanitizing it to be honest.

If we where to sanitize the version number that already exists in `plugin-name.php` then we should start sanitizing everything and that's not Health Check, thing, if there was a sanitize to be made it should be made on the core instead for this in my opinion and return a clean number.

If there's a plugin installed that doesn't adhere to the WordPress standards then you're probably having a bigger issue already than just a wrong version number in the file.

Reply:
Yes, but we should always sanitize lately. If you use PHP Codesniffer WP Rules, it should be hint on this var for this topic.

Reply:
It didn't show an error here using WordPress-Core, but @Clorith informed me that we either way use `esc_html()` on https://github.com/WordPress/health-check/blob/785140a860252ff2da827f3f9010bbc16f6411ea/src/pages/debug-data.php#L194 before showing any of these vars. That was something that I didn't include on my previous comment.

If you guys feel there's a need to double check all vars etc I'll be happy to come up with a new PR cross-checking everything ^_^

Reply:
No, I double check is not necessary or helpful. Im feel with you in this context.